docs/what-is-rocm.rst

```diff
@@ -26,57 +29,105 @@ ROCm project list
 ROCm consists of the following projects. For information on the license associated with each project,
 see :doc:`ROCm licensing <./about/license>`.
 
+Compilers
```

Comment:
Can we organize the tables, just like the graphic?  From Top -> Bottom, so it's move intuitive for users to find what they're looking for?

So Compilers move between Tools and Runtime.  

Tools
Compilers
Runtime